ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -934,6 +927,14 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     // from the same VC.
     _currentStepViewController = viewController;
     
+    NSString *progressLabel = nil;
+    if ([self shouldDisplayProgressLabel]) {
+        ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
+        if (progress.total > 0) {
+            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,ORKLocalizedStringFromNumber(@(progress.current+1)), ORKLocalizedStringFromNumber(@(progress.total))];
+        }
+    }
+    
```

Comment:
Just to clarify, you just moved this block of code, right? There wasn't any functionality changes?